File: BackEnd/src/main/java/com/Checkerly/BackEnd/repository/OrganizerRepository.java
package com.Checkerly.BackEnd.repository;

import java.util.Optional;
import org.springframework.data.mongodb.repository.MongoRepository;

import com.Checkerly.BackEnd.domain.Organizer;

public interface OrganizerRepository extends MongoRepository<Organizer, String>{
     Optional<Organizer> findByEmail(String email);
}

File: BackEnd/src/main/kotlin/com/Checkerly/BackEnd/repository/OrganizerRepository.kt
package com.Checkerly.BackEnd.repository

import com.Checkerly.BackEnd.domain.Organizer
import org.springframework.data.mongodb.repository.MongoRepository
import java.util.Optional

interface OrganizerRepository : MongoRepository<Organizer?, String?> {
    fun findByEmail(email: String?): Optional<Organizer?>?
}
